Fix for issue #53 - Wrong path for node_modules/generator-jhipster

Ebsan opened the pull request:
Used the fixes stated by @mabliedana and tested with JHipster 6.9.0.

AdrienHorgnies commented:
I don't think it's related to your code addition but the CI fails, so I can't trust the PR.

Can you look and fix the Travis build ? I suppose Travis has changed the java package availability and we must change it. Maybe switch to version 11 ? I don't know which version JHipster uses.

Ebsan commented:
@AdrienHorgnies Okay....so after a lot of back and forth I think this is building and testing correctly now. I've updated the version of Jhipster and removed some redundant Travis commands for the jdk.

AdrienHorgnies commented:
@Ebsan Very nice of you.

Any reason you deleted the `yarn.lock` file ? You can regenerate the file but it's supposed to be committed to freeze the dependencies and ensure reproducibility.
